stripe/api_resources/charge.py

```diff
@@ -4,8 +4,10 @@
 from stripe.api_resources.abstract import CreateableAPIResource
 from stripe.api_resources.abstract import UpdateableAPIResource
 from stripe.api_resources.abstract import ListableAPIResource
+from stripe.api_resources.abstract import custom_method
 
 
+@custom_method("capture", "post")
```

Comment:
Kind of like in Ruby, can we use the named variant for `http_verb` and `http_path` for a little extra clarity?

Reply:
Sure! We can't force it since Python handles keyword args differently, but we can just do it by convention when using `@custom_method`.

Reply:
Ah yes, that's what I meant (by convention). Let's do it!